content/source/docs/cloud/api/workspaces.html.md

```diff
@@ -100,7 +100,6 @@ _With a VCS repository_
         "identifier": "skierkowski/terraform-test-proj",
         "oauth-token-id": "ot-hmAyP66qk2AMVdbJ",
         "branch": "",
-        "default-branch": true
```

Comment:
Nit: all other examples don't have a trailing comma on the last line.

Reply:
Good call! I'll fix that, thanks!